craft_providers/__init__.py

```diff
@@ -17,8 +17,6 @@
 
 """Craft Providers base package."""
 
-__version__ = "1.13.0"
```

Comment:
Recommendation: Add this instead. https://github.com/canonical/starbase/blob/main/starcraft/__init__.py#LL18C1-L27C28